site/src/sphinx/server-basics.rst

```diff
@@ -45,13 +48,26 @@ Even if we opened a port, it's of no use if we didn't bind any services to them.
 
 .. code-block:: java
 
-    import com.linecorp.armeria.common.HttpRequest;
-    import com.linecorp.armeria.common.HttpResponseWriter;
-    import com.linecorp.armeria.common.HttpStatus;
-    import com.linecorp.armeria.common.MediaType;
+    import static com.linecorp.armeria.common.SessionProtocol.HTTP;
```

Comment:
We didn't put this import statement on purpose because it is already stated in the above example. :-)